test/input/illegal/declarations.sim

```diff
@@ -5,21 +5,21 @@ end
 extern t : int;
 extern V : set{E}(t);
 
-%%% bad-lattice-link-set-type-1
+%%% bad-grid-link-set-type-1
 element E
 end
 
 extern Q : set{E};
 extern P : set{E}(Q);
-extern S : lattice[1]{E}(P);
+extern S : grid{E}[1](P);
```

Comment:
Reverted to the previous format, since I agree that seems to match the semantics a little better.